src/components/App.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Link, Route, Redirect } from 'react-router-dom';
+import { Button } from 'reactstrap';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Route exact={true} path="/" component={TopPage} />
```

Comment:
In the near future, we replace this component with new component(container).
I just set frame of each page.